opm/grid/cpgrid/CpGrid.cpp

```diff
@@ -327,8 +349,8 @@ CpGrid::scatterGrid(EdgeWeightMethod method,
 
             // Partitioning given externally
             std::tie(computedCellPart, wells_on_proc, exportList, importList, wellConnections) =
-                cpgrid::createListsFromParts(*this, wells, possibleFutureConnections, nullptr, input_cell_part,
-                                                   true);
+                cpgrid::createListsFromParts(*this, wells, possibleFutureConnections, nullptr /*transmissibilities*/, input_cell_part,
+                                             true /*allowDistributedWells*/, nullptr /*gridAndWells*/, level);
```

Comment:
This is a nice change. How to do this is a matter a taste. At other places we do it like this:
```suggestion
                cpgrid::createListsFromParts(*this, wells, possibleFutureConnections, /* transmissibilities = */ nullptr, input_cell_part,
                                              /* allowDistributedWells = */ true,r /* gridAndWells = */ nullpt, level);
```
Please sure to put spaces near the asterisks.